tour-jshell/ranges-splits2.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Ranges and Splits
+---
+
+A [Range] is a specified group of `Key`s. There are many ways to create a `Range`.  Here are a few examples:
+
```

Comment:
It looks like some of the pages are the same as the Mini tour? Might be able to reuse the pages between the tours.

Reply:
Although a couple of pages contain very similar content, there were several formatting changes that cause them to differ. It may be possible to replace the older tour page with the updated jshell tour page. Not sure how difficult that would be. Can look into it later.